experimenter/experimenter/cirrus/middleware.py

```diff
@@ -0,0 +1,20 @@
+class CirrusMiddleware:
+    """
+    A middleware that provides experiment enrollment information via nimbus.
+    """
+
+    def __init__(self, get_response):
+        self.get_response = get_response
+        self.cirrus_url = settings.CIRRUS_URL
+
+    def __call__(self, request):
+        request.cirrus = {}
```

Comment:
I would actually put `None` here so we can clearly dilineate whether a Cirrus response was received.